packages/swapper/src/swappers/JupiterSwapper/utils/helpers.ts

```diff
@@ -0,0 +1,72 @@
+import { type ChainId, fromAssetId } from '@shapeshiftoss/caip'
+import type { Result } from '@sniptt/monads'
+import type { AxiosResponse } from 'axios'
+
+import type { SwapErrorRight } from '../../../types'
+import type { QuoteResponse } from '../models/QuoteResponse'
+import type { SwapInstructionsResponse } from '../models/SwapInstructionsResponse'
+import { jupiterSupportedChainIds } from './constants'
+import { jupiterService } from './jupiterService'
+
+export const isSupportedChainId = (chainId: ChainId): chainId is ChainId => {
+  return jupiterSupportedChainIds.includes(chainId)
+}
+
+const JUPITER_TRANSACTION_MAX_ACCOUNTS = 54
+
+type GetJupiterQuoteArgs = {
+  apiUrl: string
+  sourceAsset: string
+  destinationAsset: string
+  commissionBps: string
+  amount: string
+  slippageBps?: string
+}
+
+type GetJupiterSwapArgs = {
+  apiUrl: string
+  fromAddress: string
+  rawQuote: unknown
+  toAddress?: string
```

Comment:
q: do we want to make this required and always pass it in (although maybe undefined) for paranoia?